readme.md

```diff
@@ -61,6 +61,7 @@ $ np --help
     --no-yarn           Don't use Yarn
     --contents          Subdirectory to publish
     --no-release-draft  Skips opening a GitHub release draft
+    --test-script       Name of npm run script to run tests before publishing [default: test]
```

Comment:
Using `[` is inconsistent with `--branch`.

Reply:
changed it for consistency @sindresorhus